include/library/spdm_lib_config.h

```diff
@@ -310,8 +310,13 @@
 #ifndef LIBSPDM_TRANSPORT_ADDITIONAL_SIZE
 #define LIBSPDM_TRANSPORT_ADDITIONAL_SIZE    64
 #endif
+
+/* SENDER_RECIEVE buffer is the buffer for building request/response.
+ * So its size don't depend on data transfer size. Use LIBSPDM_MAX_MESSAGE_BUFFER_SIZE
+ * instead.
+ */
 #ifndef LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE
-#define LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE (LIBSPDM_DATA_TRANSFER_SIZE + \
+#define LIBSPDM_SENDER_RECEIVE_BUFFER_SIZE (LIBSPDM_MAX_MESSAGE_BUFFER_SIZE + \
```

Comment:
I dont think it is right fix to change MACRO here.

I think the caller need input large_message_buffer instead of send/receive buffer to hold the response.

Reply:
I suggest we drop this change in this PR, and create a new one, we can discuss there.